src/main/java/nc/students/ayaz/model/Video.java

```diff
@@ -1,18 +1,27 @@
-package nc.students.ayaz;
+package nc.students.ayaz.model;
 
 import cobertura.IgnoreDuringCodeCoverage;
 
 import java.util.ArrayList;
 import java.util.List;
 
-class Video {
+public class Video {
 
     private String name;
```

Comment:
That's not what I meant :)
If i were talking about the class itself I would comment on the line 8.
I was talking about making class member fields final - it is a best practice.
Making a class itself final (which you might do in order to prevent someone inheriting from it) is not very widespread and I would rather say that its not worth it ;)